worlds/osrs/LogicCSV/items_generated.py

```diff
@@ -0,0 +1,43 @@
+"""
+This file was auto generated by LogicCSVToPython.py
+"""
+from BaseClasses import ItemClassification
+from worlds.osrs.Items import ItemRow
```

Comment:
```suggestion
from ..Items import ItemRow
```

Need to use relative imports for stuff in your apworld